tests/fixtures/optimizer/tpc-ds/tpc-ds.sql

```diff
@@ -6589,9 +6609,9 @@ WITH "cs_or_ws_sales" AS (
     ON "customer_address"."ca_county" = "store"."s_county"
     AND "customer_address"."ca_state" = "store"."s_state"
   JOIN "_u_0" AS "_u_0"
-    ON "date_dim"."d_month_seq" >= "_u_0"."_col_0"
+    ON "date_dim"."d_month_seq" >= MAX("_u_0"."_col_0")
```

Comment:
i don't think this one is correct

Reply:
Made a few changes to address this, let me know when you take another look